templates/frontend/objects/monograph_summary.tpl

```diff
@@ -11,25 +11,29 @@
  * @uses $isFeatured bool Is this a featured monograph?
  *}
 <div class="obj_monograph_summary{if $isFeatured} is_featured{/if}">
-	<a href="{url page="catalog" op="book" path=$monograph->getBestId()}" class="cover">
-		{assign var="coverImage" value=$monograph->getCurrentPublication()->getLocalizedData('coverImage')}
-		<img
-			src="{$monograph->getCurrentPublication()->getLocalizedCoverImageThumbnailUrl($monograph->getData('contextId'))}"
-			alt="{$coverImage.altText|escape|default:''}"
-		>
-	</a>
-	{if $monograph->getSeriesPosition()}
-		<div class="seriesPosition">
-			{$monograph->getSeriesPosition()|escape}
+		<a href="{url page="catalog" op="book" path=$monograph->getBestId()}" class="cover">
+			{assign var="coverImage" value=$monograph->getCurrentPublication()->getLocalizedData('coverImage')}
+			<img
+				src="{$monograph->getCurrentPublication()->getLocalizedCoverImageThumbnailUrl($monograph->getData('contextId'))}"
+				alt="{$coverImage.altText|escape|default:''}"
+			>
+		</a>
+		<div class="wrapper">
+			{if $monograph->getSeriesPosition()}
+				<div class="seriesPosition">
+					{$monograph->getSeriesPosition()|escape}
+				</div>
+			{/if}
+			<{$heading} class="title">
+				<a href="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="book" path=$monograph->getBestId()}">
+					{$monograph->getLocalizedFullTitle()|escape}
+				</a>
+			</{$heading}>
+			<div class="author">
+				{$monograph->getAuthorOrEditorString()|escape}
+			</div>
+			<div class="date">
+				{$monograph->getDatePublished()|date_format:$dateFormatLong}
+			</div>
```

Comment:
It looks like maybe there's some work in here for another issue? I think you can get rid of the `wrapper` here and in the CSS above.

Reply:
I'm finding that the `<h2>` gets pushed back to the left, depending on whether there is a cover image or not. Adding a `margin-left: @base...` causes one to move back to the left (or right), or align just right with the rest of the text. What's the best way to approach this?

Reply:
Can you provide some screenshots? In OMP, if there's no cover image, there should usually be a default image that gets displayed.

Reply:
<img width="855" alt="Screen Shot 2020-04-07 at 08 45 26" src="https://user-images.githubusercontent.com/38994650/78670711-2b09ec00-78ac-11ea-8a7d-7c850a98abfc.png">

Maybe it's just the way mine is set up?

Reply:
That looks to me like the second book (on the right) has a cover image but the image isn't showing up properly. Maybe the one on the left is happening because of the additional CSS added here? https://github.com/pkp/omp/pull/795/files#diff-83194493df7b9f27c840498b649933f0R44-R50

Reply:
This one's been resolved in https://github.com/pkp/omp/pull/795/commits/556766a0160fd886258697ff7445c828207de165